### PR TITLE
data:AnshikGupta

### PR DESCRIPTION
**Pull request by guptaanshik1**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5578"></a>

### CODE REVIEW

File: data/AnshikGupta.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/anshik-gupta-5a99ab192/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/guptaanshik1",
+      "icon": "FaGithub"
+    },
```
Comment (imbeshat) [inline data/AnshikGupta.json:15]:
```suggestion
    }
```

**Review by imbeshat — COMMENTED** [review]
Also the filename should be as ```YourGithubUsername.json```

File: data/AnshikGupta.json (hunk 1 above)
Comment (guptaanshik1) [reply data/AnshikGupta.json]: I have updated it

**Review by khushalarora — APPROVED** [review]
Thank you for contributing  :tada:

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**